scripts/split-paired-reads.py

```diff
@@ -37,16 +38,16 @@ def get_parser():
     left- and right- reads separated. This reformats the former to the latter.
 
     The directory into which the left- and right- reads are output may be
-    specified using :option:`-o`/:option:`--output-dir`. This directory will be
+    specified using :option:`-d`/:option:`--output-dir`. This directory will be
     created if it does not already exist.
 
     Alternatively, you can specify the filenames directly with
     :option:`-1`/:option:`--output-first` and
     :option:`-2`/:option:`--output-second`, which will override the
-    :option:`-o`/:option:`--output-dir` setting on a file-specific basis.
+    :option:`-d`/:option:`--output-dir` setting on a file-specific basis.
 
-    :option:`-p`/:option:`--force-paired` will require the input file to
-    be properly interleaved; by default, this is not required.
+    :option:`-0`/:option:'`--output-orphans` will allow broken-paired format,
```

Comment:
On Tue, Aug 04, 2015 at 09:50:34AM -0700, Michael R. Crusoe wrote:
> >  
> > -    :option:`-p`/:option:`--force-paired` will require the input file to
> > -    be properly interleaved; by default, this is not required.
> > +    :option:`-0`/:option:'`--output-orphans` will allow broken-paired format,
> 
> extra extraneous single quote after the 2nd `:option:`

how do you find this stuff!? fixed :)